srcs/runners_manager/vm_creation/aws/schema.py

```diff
@@ -1,5 +1,7 @@
 from marshmallow import Schema, fields
 
+class AwsCloudConfig(Schema):
+    aws_tags = fields.Dict(keys=fields.Str(), values=fields.Str())
```

Comment:
Seems a bit repetitive to mention `aws` in a config that is specific to `aws`. I wouldn't mind the config to look like this at the end:

```yaml
cloud_name: aws
cloud_config:
  tags:
    my_tag: value
```

```suggestion
    tags = fields.Dict(keys=fields.Str(), values=fields.Str())
```
If you want to differentiate `tags` of github runners and aws tags you can use `ec2_tags` that's a legitimate specification.

And please ensure the value is set as optional (most likely is already the case, just want to double check)

Reply:
I check and change everything once I get it to work! Thanks for the comment!

Reply:
Done, everything's good! It is optional, I just checked